dietpi/dietpi-software

```diff
@@ -10385,7 +10392,27 @@ _EOF_
 			fi
 
 		fi
+#---------------------------------
+		software_id=142 # microk8s
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# APT deps
+			G_AGI snapd
+		#install
+		G_EXEC snap install microk8s --classic
```

Comment:
I guess the classic mode is required instead of confinement/jailed mode to allow communication with its containers? Did some reading: https://snapcraft.io/docs/snap-confinement